community.json

```diff
@@ -639,6 +639,15 @@
       "tags": ["ca", "crow", "sequencer"],
       "origin": "lines"
     },
+    {
+      "project_name": "less_concepts_3",
+      "project_url": "https://github.com/dndrks/less_concepts_3",
+      "author": "jai lai bookie (Dan_Derks) & vicimity (Linus Schrab)",
```

Comment:
also, dang, i forgot i was using that name lol...let's make this:
 "author": "@dan_derks & vicimity (Linus Schrab)",